wp-content/themes/pub/wporg-learn-2020/archive-wporg_workshop.php

```diff
@@ -25,7 +25,6 @@
 	<section>
 		<div class="row align-middle between section-heading section-heading--with-space gutters">
 			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
```

Comment:
I don't think we need to handle a row layout here anymore

```suggestion
		<div class="section-heading section-heading--with-space">
			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
```

Reply:
`col-12` can be removed from line 31 also

Reply:
OK - I copied this from the courses archive page, so I assume it was all needed, but happy to remove it if it's not necessary here.